README.md

```diff
@@ -49,58 +49,55 @@ messenger.on('text', ({reply, text}) => {
 messenger.start();
 ```
 
+<<<<<<< HEAD
```

Comment:
Yeah something did not push correctly after I thought I resolved them... there were dup'ed tests also. Fixing this now.